### PR TITLE
doc: add unstable

### PR DESCRIPTION
**Pull request by xiang90**
@heyitsanthony @gyuho 

Shall we call them experimental than unstable? unstable sounds scary... We want it sounds somewhat scary, but we also encourage people to try them out.

### CODE REVIEW

**Comment by heyitsanthony** [comment]
experimental sounds more like "this is subject to change" whereas unstable is more "we're ironing out the bugs"

**Comment by xiang90** [comment]
@heyitsanthony OK. I feel experimental is more accurate now. The Auth API, for example, might be slightly changed until becoming stable.

**Comment by gyuho** [comment]
Oh sorry, should have waited for this to be merged :(

**Comment by xiang90** [comment]
i will rebase.

**Comment by gyuho** [comment]
thanks.

**Comment by xiang90** [comment]
@heyitsanthony Looks good now?

File: Documentation/dev-guide/experimental_apis.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Experimental APIs and features
+
+For the most part, the etcd project is stable, but we are still moving fast! We believe in the release fast philosophy. We want to get early feedback on features still in development and stabilizing. Thus, there are, and will be more, experimental features and APIs. We plan to improve these features based on the early feedback from community, or abandon them if there is little interest, in the next few releases. If you are running a production system, please do not rely on any experimental features or APIs.
```
Comment (heyitsanthony) [inline Documentation/dev-guide/experimental_apis.md]: s/from community/from the community/

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**